sklearn/metrics/_plot/confusion_matrix.py

```diff
@@ -108,11 +109,16 @@ def plot(self, include_values=True, cmap='viridis',
                     ha="center", va="center",
                     color=color)
 
+        if self.display_labels is None:
+            display_labels = np.arange(n_classes)
+        else:
+            display_labels = self.display_labels
```

Comment:
Do we like that kind of idiom? Just wondering

```py
display_labels = self.display_labels or np.arange(n_classes)
```

Reply:
We can not do this because `self.display_labels` can be a numpy array.